src/main/java/ch/njol/skript/Skript.java

```diff
@@ -550,32 +553,24 @@ public void onEnable() {
 			@Override
 			public void run() {
 				assert Bukkit.getWorlds().get(0).getFullTime() == tick;
-				
-				// Load hooks from Skript jar
-				try {
-					try (JarFile jar = new JarFile(getFile())) {
-						for (JarEntry e : new EnumerationIterable<>(jar.entries())) {
-							if (e.getName().startsWith("ch/njol/skript/hooks/") && e.getName().endsWith("Hook.class") && StringUtils.count("" + e.getName(), '/') <= 5) {
-								final String c = e.getName().replace('/', '.').substring(0, e.getName().length() - ".class".length());
-								try {
-									Class<?> hook = Class.forName(c, true, getClassLoader());
-									if (Hook.class.isAssignableFrom(hook) && !Modifier.isAbstract(hook.getModifiers()) && isHookEnabled((Class<? extends Hook<?>>) hook)) {
-										hook.getDeclaredConstructor().setAccessible(true);
-										hook.getDeclaredConstructor().newInstance();
-									}
-								} catch (ClassNotFoundException ex) {
-									Skript.exception(ex, "Cannot load class " + c);
-								} catch (ExceptionInInitializerError err) {
-									Skript.exception(err.getCause(), "Class " + c + " generated an exception while loading");
-								} catch (Exception ex) {
-									Skript.exception(ex, "Exception initializing hook: " + c);
-								}
-							}
-						}
+
+				// Hooks
+				Map<Class<? extends Hook>, Supplier<Hook>> hooks = ImmutableMap.of(
+						GriefPreventionHook.class, GriefPreventionHook::new,
+						PreciousStonesHook.class, PreciousStonesHook::new,
+						ResidenceHook.class, ResidenceHook::new,
+						WorldGuardHook.class, WorldGuardHook::new,
+						VaultHook.class, VaultHook::new);
+
+				for (Entry<Class<? extends Hook>, Supplier<Hook>> entry : hooks.entrySet()) {
+					if (!isHookEnabled(entry.getKey()))
+						continue;
+
+					try {
+						entry.getValue().get();
+					} catch (Exception e) {
```

Comment:
```suggestion
					} catch (Exception ex) {
```

Reply:
I disagree with all the `e` -> `ex` [Google](https://google.github.io/styleguide/javaguide.html) uses `e` too and so do we--there are 25 matches for `ex` and 100+ for `e`.

Reply:
IMO `e` is misleading as it was used for event a lot, either `exception` or `ex`, just better than `e`

Reply:
Events should be `event` not `e`, there's a consensus to use `e` for exceptions

Reply:
If you do want to make this change there should be some discussion with multiple contributors